sql/hive-thriftserver/v1.2/src/main/java/org/apache/hive/service/auth/HiveAuthFactory.java

```diff
@@ -291,14 +291,14 @@ public static TServerSocket getServerSSLSocket(String hiveHost, int portNum, Str
     TServerSocket thriftServerSocket =
         TSSLTransportFactory.getServerSocket(portNum, 0, serverAddress.getAddress(), params);
     if (thriftServerSocket.getServerSocket() instanceof SSLServerSocket) {
-      List<String> sslVersionBlacklistLocal = new ArrayList<String>();
-      for (String sslVersion : sslVersionBlacklist) {
-        sslVersionBlacklistLocal.add(sslVersion.trim().toLowerCase(Locale.ROOT));
+      List<String> sslVersionsExcludedLocal = new ArrayList<String>();
+      for (String sslVersion : sslVersionsExcluded) {
+        sslVersionsExcludedLocal.add(sslVersion.trim().toLowerCase(Locale.ROOT));
       }
       SSLServerSocket sslServerSocket = (SSLServerSocket) thriftServerSocket.getServerSocket();
       List<String> enabledProtocols = new ArrayList<String>();
       for (String protocol : sslServerSocket.getEnabledProtocols()) {
-        if (sslVersionBlacklistLocal.contains(protocol.toLowerCase(Locale.ROOT))) {
+        if (sslVersionsExcludedLocal.contains(protocol.toLowerCase(Locale.ROOT))) {
```

Comment:
Changes here (and in the other `sql/hive-thriftserver` files) shouldn't be included in this PR. Previously I was suggested not to adjust these since they are a direct copy of Hive files -- and either way it's not relevant to the excludeOnFailure feature.

Reply:
ok if its an exact copy I'll revert, otherwise I don't really see the harm in renaming.

Reply:
Best I can tell, it is copied from here: https://github.com/apache/hive/blob/branch-1.2/service/src/java/org/apache/hive/service/auth/HiveAuthFactory.java